internal/lib/stats/stats_server.go

```diff
@@ -185,7 +185,11 @@ func (ss *StatsServer) writableLayerForContainer(container *oci.Container) (*typ
 		Timestamp: time.Now().UnixNano(),
 		FsId:      &types.FilesystemIdentifier{Mountpoint: container.MountPoint()},
 	}
-	driver, err := ss.Store().GraphDriver()
+	store, err := ss.Store().GetStoreForContainer(container.ID())
+	if err != nil {
+		return writableLayer, errors.Wrapf(err, "Unable to get store for container %s", container.ID())
```

Comment:
ultra nit, we start errors with lowercase usually:
```suggestion
		return writableLayer, errors.Wrapf(err, "unable to get store for container %s", container.ID())
```